3-JSKata/1-objects-and-arrays/kata.js

```
@@ -52,20 +90,45 @@ function getValues(obj) {}
 // Tip: consider JavaScript's Rest parameters
-function makeArrayOfItems() {}
+function makeArrayOfItems(...item) {
+  return item // Using the rest Parameter, it captures everything and stores it in an array
+}
 
 // hasItem should return true if `item` is present in `arr` at least once,
 // otherwise it should return false.
 // Tip: there is an array function that makes this straightforward
-function hasItem(arr, item) {}
+function hasItem(arr, item) {
+  for (let i = 0; i < arr.length; i++) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)